### PR TITLE
Make date time picker columns grow when label is too wide

### PR DESCRIPTION
**Pull request by ariaieboy**
before in fa lang
image

after in fa lang
image

after in en lang
image

### CODE REVIEW

**Comment by danharrin** [comment]
Pretty sure there is a reason we did it like that before

**Comment by zepfietje** [comment]
> Pretty sure there is a reason we did it like that before

Yeah, probably for min width of the date buttons.

**Comment by danharrin** [comment]
Is the min width the `spacing.7` bit?

**Comment by zepfietje** [comment]
Yeah I suppose.

**Review by zepfietje — CHANGES_REQUESTED** [review]
Please make it so that the default width is still used, and the buttons only grow when the label is too wide, @ariaieboy.

File: packages/forms/resources/views/components/date-time-picker.blade.php [hunk 1]
```diff
@@ -155,7 +155,7 @@
                         'fi-fo-date-time-picker-panel absolute z-10 rounded-lg bg-white p-4 shadow-lg ring-1 ring-gray-950/5 dark:bg-gray-900 dark:ring-white/10',
                     ])
                 >
-                    <div class="grid gap-y-3">
+                    <div class="gap-y-3">
```
Comment (danharrin) [inline packages/forms/resources/views/components/date-time-picker.blade.php]: Gap only works with grid or flex elements, right?

Comment (ariaieboy) [reply packages/forms/resources/views/components/date-time-picker.blade.php]: I've only changed line 197 what is this codes 🤔

Comment (danharrin) [reply packages/forms/resources/views/components/date-time-picker.blade.php]: Please check the full git diff

**Comment by zepfietje** [comment]
Tested the change and it seems to keep the original behavior when labels aren't too wide.
Thanks, @ariaieboy!

**Review by zepfietje — APPROVED** [review]

**Pull request merged**